### PR TITLE
creating a pytorch template for a GPU enabled packageset

### PR DESCRIPTION
**Pull request by zeryx**

### CODE REVIEW

File: templates/pytorch-1.0.0/src/__ALGO__.py [hunk 1]
```diff
@@ -1,53 +1,15 @@
...
-   return output
+    A = th.randn(input).cuda()
+    B = th.randn(input).cuda()
+    C = th.dot(A, B)
+    random_number = C.cpu().numpy().tolist()
+    return "with a random vector shape of {}, here is your random number {}".format(input, str(random_number))
 
 
-model = get_model(client, MODEL_PATH)
+print(apply(56))
```
Comment (pmcq) [inline templates/pytorch-1.0.0/src/__ALGO__.py]: nit: probably don't want this

**Review by pmcq — COMMENTED** [review]
seems fine to me too - maybe want a comment saying the expected input type is an int or something

**Review by pmcq — APPROVED** [review]

**Comment by kennydaniel** [comment]
Holding off on approving this... probably want same behavior as tf example with matrix dot?

File: templates/pytorch-1.0.0/src/__ALGO__.py (hunk 1 above)
Comment (zeryx) [reply templates/pytorch-1.0.0/src/__ALGO__.py]: oh yea good point, will also remove that global debugging print

**Pull request merged**